test/CMakeLists.txt

```diff
@@ -75,11 +74,16 @@ target_link_system_libraries(
 enable_testing()
 add_test(NAME main COMMAND main)
 
+### to get CMAKE_INSTALL_INCLUDEDIR
+# NOTE: enable at least one language before including GNUInstallDirs.
+include(GNUInstallDirs)
```

Comment:
Should we add an option for this? I think including it by default could break some code out there.

Reply:
I may not imagine that would be a Problem, but there is an other way to do only if really needed.